workloads/mastervertical.yml

```diff
@@ -12,6 +12,18 @@
   vars:
     workload_job: "mastervertical"
   tasks:
+    # Get workload start time
```

Comment:
Since you have cleanly implemented prometheus as a separate role, can we move this there?